xla/backends/gpu/runtime/dynamic_slice_thunk.cc

```diff
@@ -47,14 +48,30 @@ limitations under the License.
 namespace xla {
 namespace gpu {
 
+namespace {
+
+// Indvar is a thread-local map that stores the induction variable for each
+// dynamic slice thunk. The usage of threadlocal in this context is similar to
+// `LoopCounters` in while_thunk.cc (b/343294327).
+Literal& Indvar(DynamicSliceThunk* thunk) {
```

Comment:
These definitions are also in another change, where I commented on them & their documentation. Where should I review their introduction?

Either way, I'd like to see better documentations of why this is an appropriate implementation, and what this relies on :)